src/apic-extension/azext_apic_extension/tests/latest/test_integration_commands.py

```diff
@@ -64,7 +64,25 @@ def test_integration_create_apim(self):
 
     @ResourceGroupPreparer(name_prefix="clirg", location="centraluseuap", random_name_length=32)
     @ApicServicePreparer()
-    @ApimServicePreparer()
     def test_integration_create_aws(self):
-        # TODO: add codes for AWS integration test
-        pass
+        if self.is_live:
+            # prepare test data
+            self.kwargs.update({
+                'integration_name': self.create_random_name(prefix='cli', length=8),
+                'usi_id': USERASSIGNED_IDENTITY,
+                'access_key_link': AWS_ACCESS_KEY_LINK,
+                'secret_access_key_link': AWS_SECRET_ACCESS_KEY_LINK,
+                'aws_region': AWS_REGION
+            })
+
+            # Attach user assigned identity with access to AWS KeyVault to API Center 
+            self.cmd('az apic update --name {s} -g {rg} --identity {{type:UserAssigned,user-assigned-identities:{{{usi_id}}}}}')
```

Comment:
minor: we may update `ApicServicePreparer` to enable user assigned identity when needed, so you don't need to repeat this line every time. Maybe something like `@ApicServicePreparer(user_assigned_identity=usi_id)`

Reply:
sounds good